storage/Devicegraph.h

```diff
@@ -199,32 +213,89 @@ namespace storage
 	 */
 	bool holder_exists(sid_t source_sid, sid_t target_sid) const;
 
+	/**
+	 * Clear the devicegraph.
```

Comment:
Maybe mention that this invalidates all pointers to objects in the devicegraph?
It _should_ be clear, but we had bugs about this before...